docs/getting_started.md

```diff
@@ -52,7 +52,9 @@ Also available in Mac, Linux and WSL Homebrew:
 ```bash
 brew install argocd
 ```
-
+!!! note
+    To ensure proper functionality, the Argo CD CLI requires HTTP/2 support. Verify that your CLI is set up to use HTTP/2, especially if connecting to an Argo CD server over HTTPS. Check your system’s configuration or proxy settings if issues arise.
```

Comment:
> that your CLI is set up to use HTTP/2

This is a little confusing to me. You point out that the cli uses http/2 by default. I *think* we should say that the gateway / ingress controller should be configured to use http/2?

Reply:
@rumstead  Better now?